temp-draft-4.md

```diff
@@ -311,7 +313,7 @@ Transforms are only used in the Presentation API when the SpecificResource is th
 
 ### Relative Rotation
 
-It is useful to be able to rotate a light or camera resource such that it is facing another object or point in the Scene, rather than calculating the angles within the Scene's coordinate space. This is accomplished with a property called `lookAt`, on DirectionalLight, SpotLight and all Cameras. The value of the property is either a PointSelector or the URI of an Annotation which paints something into the current Scene.
+It is useful to be able to rotate a light or camera resource such that it is facing another object or point in the Scene, rather than calculating the angles within the Scene's coordinate space. This is accomplished with a property called `lookAt`, valid on DirectionalLight, SpotLight, and all Cameras. The value of the property is either a PointSelector or the URI of an Annotation which paints something into the current Scene.
 
 If the value is a PointSelector, then the light or camera resource is rotated around the x and y axes such that it is facing the given point. If the value is an Annotation which targets a point via a PointSelector, URI fragment or other mechanism, then the direction the resource is facing that point.
```

Comment:
The last bit here is unclear -- "then the direction the resource is facing that point."